VISION.md

```diff
@@ -1,12 +1,12 @@
 # OpenEBS Umbrella Project Vision
 
 > [!Important]
-> OpenEBS is an "umbrella" project,  composed as a federation of individual sub projects (repositories). The umbrella project, every sub project, repository/repo and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
+> OpenEBS is an "umbrella" project,  composed as a federation of individual sub-projects (repositories). The umbrella project, every sub-project, repository/repo, and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
```

Comment:
the extra , should be removed ("and" negates need for , in most cases)

Reply:
I used a AI to detect the issues and it seems to suggest a comma. Should I remove it?

Reply:
Done.